clients/upsclient.c

```diff
@@ -315,22 +310,35 @@ int upscli_init(int certverify, const char *certpath,
 	}
 	
 #ifdef WITH_OPENSSL
+
+	SSL_load_error_strings();
 	
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
 	SSL_library_init();
-	SSL_load_error_strings();
 
-	ssl_method = TLSv1_client_method();
+	ssl_ctx = SSL_CTX_new(SSLv23_client_method());
+#else
+	OPENSSL_init_ssl(0, NULL);
```

Comment:
This call is not necessary, as library initialization is done automatically now.

Reply:
Agree